Aztec/Classes/Extensions/NSAttributedString+HTMLInitializer.swift

```diff
@@ -3,14 +3,19 @@ import UIKit
 
 extension NSAttributedString {
 
-    convenience init(withHTML html: String, defaultAttributes: [String: Any]) {
-
+    convenience init(
+        withHTML html: String,
+        defaultAttributes: [String: Any],
+        postProcessingHTMLWith htmlTreeProcessor: HTMLTreeProcessor? = nil) {
```

Comment:
@jleandroperez - Thoughts about renaming this to something else?  Maybe `processingHTMLTreeWith`?

Reply:
Other options:

`postProcessingWith`
`processingTreeWith`
`using HTMLTreeProcessor` -- this one looks nice.

Reply:
As discussed, let's leave this untouched for now. We'll get back to it in another PR!